code/e2e-tests/manager.spec.ts

```diff
@@ -107,7 +107,7 @@ test.describe('Manager UI', () => {
       const sbPage = new SbPage(page, expect);
       await sbPage.navigateToStory('example/button', 'primary');
       await expect(page.getByRole('button', { name: 'Open in editor' })).toBeVisible();
-      await page.getByRole('button', { name: 'Share' }).click();
+      await page.getByRole('button', { name: 'Share', exact: true }).click();
```

Comment:
I gotta say that your fix is good, but the selector that was there was doomed to fail at some point (by selecting the wrong node).
And frankly, it still is.